libraries/teams-scenarios/fileUpload/src/fileUploadBot.ts

```diff
@@ -73,7 +73,7 @@ export class FileUploadBot extends TeamsActivityHandler {
         const path = require('path');
         const filePath = path.join('files', context['filename']);
         const stats = fs.statSync(filePath);
-        const fileSizeInBytes = stats['size']; 
+        // const fileSizeInBytes = stats['size']; 
```

Comment:
Please remove this line if it's not necessary.

Reply:
Yea, thanks, I missed that one.